manageiq-operator/api/v1alpha1/helpers/miq-components/cr.go

```diff
@@ -51,7 +51,7 @@ func databaseVolumeCapacity(cr *miqv1alpha1.ManageIQ) string {
 
 func deployMessagingService(cr *miqv1alpha1.ManageIQ) bool {
 	if cr.Spec.DeployMessagingService == nil {
-		return false
+		return true
```

Comment:
I think we can remove the conditional completely and always return true unless we add some conditional logic to the service checking in the entrypoint script.  As is it, the orchestrator will wait foerver for the kafka if the user sets this to false, right?